terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,72 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                 = "filing-maintain" # this must match the stack name the service deploys into
+  name_prefix                = "${local.stack_name}-${var.environment}"
+  global_prefix              = "global-${var.environment}"
+  service_name               = "officer-filing-api"
+  container_port             = 8080
+  eric_port                  = "10000"
+  docker_repo                = "officer-filing-api"
+  kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
+  lb_listener_rule_priority  = 61
+  lb_listener_paths          = ["/transactions/.*/officers", "/private/transactions/.*/officers", "/transactions/.*/officers/.*"]
```

Comment:
PathPrefix(`/transactions/{transaction_id:.*}/officers`) || PathPrefix(`/transactions/{transaction_id:.*}/officers/{filing_resource_id:.*}`) || PathPrefix(`/private/transactions/{transaction_id:.*}/officers/{filing_resource_id:.*}`)) <- maybe need to update this? I'm not certain if we need .* or if * is sufficient - devs may be able to confirm

Reply:
Confirmed with dev and changed